site/content/integrate/apps/interactivity/_index.md

```diff
@@ -24,14 +24,17 @@ They are defined by:
 
 The types of fields are:
 
-| Name             | Description                                            |
-| :--------------- | :----------------------------------------------------- |
-| `text`           | Text field.                                            |
-| `static_select`  | A dropdown select with static elements.                |
-| `dynamic_select` | A dropdown select that loads the elements dynamically. |
-| `bool`           | A boolean selector represented as a checkbox.          |
-| `user`           | A dropdown with to select users.                       |
-| `channel`        | A dropdown to select channels.                         |
+| Name             | Description                                              |
+| :--------------- | :------------------------------------------------------- |
+| `text`           | Text field.                                              |
+| `static_select`  | A dropdown select with static elements.                  |
+| `dynamic_select` | A dropdown select that loads the elements dynamically.   |
+| `bool`           | A boolean selector represented as a checkbox.            |
+| `user`           | A dropdown to select users.                              |
+| `channel`        | A dropdown to select channels.                           |
+| `markdown`       | Only visible on modal forms. An arbitrary markdown text. |
+
+Markdown fields are a special field that allows you to better format your form. They will not generate any value towards the app. The content is defined in the field description.
```

Comment:
```suggestion
Markdown fields are a special field that allows you to better format your form. They will not generate any value in the form submission sent to the app. The content is defined in the field description.
```

Reply:
Why is the content defined in the description? Seems reasonable

Reply:
The only reason I considered was lack of a better place :sweat_smile: